pom.xml

```diff
@@ -250,6 +251,7 @@
                               <exclude>**/src/site/resources/**/*.txt</exclude>
                               <exclude>**/src/site/resources/**/*.json</exclude>
                               <exclude>**/.gitignore</exclude>
+                              <exclude>**/*.md</exclude>
```

Comment:
@steveblackmon Yes. I can go back and add the license to the markdown. Wasn't thinking about rat being able to pull that out of markdown.